Extension/src/nativeStrings.json

```diff
@@ -178,15 +178,35 @@
         "hint": "Refers to C++ template specializations. This is part of a description indicating there are N additional specializations of a function.  %d is replaced with that number."
     },
     "expands_to": "Expands to:",
+    "attention_label": "Attention:",
+    "author_label": "Author:",
+    "authors_label": "Authors:",
+    "bug_label": "Bug:",
+    "copyright_label" : "Copyright:",
+    "deprecated_label": "Deprecated:",
+    "date_label" : "Date:",
+    "details_label" : "Details:",
+    "exceptions_label": "Exceptions:",
+    "invariant_label": "Invariant:",
     "file_label": "File:",
+    "note_label": "Note:",
     "parameters_label": "Parameters:",
+    "pre_label": "Precondition:",
+    "post_label": "Postcondition:",
+    "remark_label": "Remark:",
+    "remarks_label": "Remarks:",
+    "result_label": "Result:",
+    "return_label": "Return:",
```

Comment:
Our loc team tends to have a lot of difficulty with strings in our area, as they aren't coders and aren't generally familiar with the concepts involved. I'd recommend providing a hint for any word even remotely ambiguous.  Maybe add hints for `Exceptions` and `Return`.